rdflib/term.py

```diff
@@ -1383,6 +1383,14 @@ def _unhexlify(value):
         value = value.encode()
     return unhexlify(value)
 
+def _parseBoolean(value):
+    new_value = value.lower()
+    if new_value == 'true' or new_value=='1':
+        return True
+    if new_value != 'false' and new_value != '0':
+        warnings.warn('Parsing weird boolean, %r does not map to True or False'%value,DeprecationWarning)
```

Comment:
could you please specify the attribute name `category=` for the warning and add spaces around the `%`.

Reply:
I've added these changes in the new commit in this pull request :)